wg-ecosystem/README.md

```diff
@@ -63,22 +63,38 @@ These repos are sorted alphabetically, their order does not reflect that any of
 
 ### Electron Organization
 
+* [`electron/algolia-indices`](https://github.com/electron/algolia-indices)
 * [`electron/apps`](https://github.com/electron/apps)
+* [`electron/asar`](https://github.com/electron/asar)
+* [`electron/docs-parser`](https://github.com/electron/docs-parser)
+* [`electron/download-stats`](https://github.com/electron/download-stats)
 * [`electron/electron-api-demos`](https://github.com/electron/electron-api-demos)
+* [`electron/electron-api-historian`](https://github.com/electron/electron-api-historian)
 * [`electron/electron-docs-linter`](https://github.com/electron/electron-docs-linter)
+* [`electron/electron-docs`](https://github.com/electron/electron-docs)
```

Comment:
nit: electron-docs should go in front of electron-docs-linter for alphabetical order